components/JournalEntryForm/index.js

```diff
@@ -31,9 +70,19 @@ export default function EntryForm({ onFormSubmit, entry }) {
     if (entry && entry.id) {
       entryObject.id = entry.id;
     }
-    onFormSubmit(entryObject);
+    onFormSubmit(entryObject, entry.id);
     router.push("/journal");
   }
+
+  const handleRemoveImage = () => {
+    setUrl("");
+    setShowWarning(true);
+    const input = document.getElementById("plantbuddyImage");
+    if (input) {
+      input.value = null;
+    }
+  };
```

Comment:
Try to avoid using the normal DOM Api, you can either receive the image inputs value from the event, or you control the input with a state.